src/App.js

```diff
@@ -14,10 +15,16 @@ function App() {
         setMovies(res.data.results);
       });
   }, []);
-  console.log(movies);
+
+  const showSearchResults = (movies) => {
+    setMovies(movies);
+  };
+
   return (
     <div className="App">
-      <MovieList movies={movies}  />
+      <NavBar passToApp={showSearchResults} />
```

Comment:
Pass to app is an ambiguous name for a prop.
Please rename this according to the other comments in the review.

```suggestion
      <NavBar ?={showSearchResults} />
```